src/python/dxpy/scripts/dx_build_app.py

```diff
@@ -35,8 +35,11 @@
 import dxpy.app_builder
 import dxpy.workflow_builder
 import dxpy.executable_builder
+from dxpy.nextflow.nextflow_utils import get_resources_subpath
 from .. import logger
+from pathlib import Path
 
+from dxpy.nextflow.nextflow_builder import *
```

Comment:
This will pollute the namespace and makes the code harder to read (I don't see immediately where some methods come from)